pkg/virt-handler/node-labeller/node_labeller.go

```diff
@@ -340,3 +340,10 @@ func (n *NodeLabeller) alertIfHostModelIsObsolete(originalNode *v1.Node, hostMod
 	n.recorder.Eventf(originalNode, v1.EventTypeWarning, "HostModelIsObsolete", warningMsg)
 	return nil
 }
+
+func (n *NodeLabeller) GetTSCCounter() *libvirtxml.CapsHostCPUCounter {
```

Comment:
Should this function be exported?

Reply:
This method  is currently exported so that the `cpu_plugin_test.go` is able to make use of it during its test `should return correct cpu models, features and tsc freqnency`, the test is removed entirely in the third commit and the `GetTSCCounter()` method changes and becomes unexported.